### PR TITLE
Quest graphics struct new/init/free and tests

### PR DESCRIPTION
**Pull request by cyruszzhou**
We have written structs for quest graphics display. We also provided new init free functions for the struct. All tests written with criterion have passed. We want to merge it to the dev branch.

### CODE REVIEW

File: graphics_sandbox/quest_graphics/quest_graphics.h [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+ * Parameters:
+ *      A pointer to a quest_graphics struct (allocated memory),
+ *      and everythiing in the quest_graphics struct
+ * 
+ * Function:
+ *      Initialize the memory allocated to the quest_graphics struct
+ *      with information provided through the parameters
+ * 
+ * Returns:
+ *      1 on success
```
Comment (philipchenaux) [inline graphics_sandbox/quest_graphics/quest_graphics.h]: Please refer to my comment on https://github.com/uchicago-cs/chiventure/pull/1494#pullrequestreview-982216190 about returning 1 from functions. You should implement functions that return either SUCCESS or FAILURE.

**Review by philipchenaux — CHANGES_REQUESTED** [review]
I'm happy with how this looks, again a minor comment on return values from functions. By the way it is good that you have included the .gitignore file as part of your PR now, that way the .DS_Store files no longer are included.

**Review by csraffaga — APPROVED** [review]
tests run and build perfectly! all pass

File: graphics_sandbox/quest_graphics/quest_graphics.h (hunk 1 above)
Comment (cyruszzhou) [reply graphics_sandbox/quest_graphics/quest_graphics.h]: Thanks! Just resolved!

**Review by philipchenaux — APPROVED** [review]
Great job, looks good to merge!

**Comment by philipchenaux** [comment]
Issue Score: Excellent

Comments:
Excellent job on this PR!

**Pull request merged**